README.md

```diff
@@ -60,6 +60,22 @@ Check the [sls invoke local command documentation](https://www.serverless.com/fr
 
 Note that to run locally, you need to export any environment variables used in code to your current environment. They can be found in the AWS Lambda configurations.
 
+#### Running the local dev database
+1. Make sure you have the following installed:
+    - Docker
+        - We recommend downloading [Docker Desktop](https://docs.docker.com/desktop/).
+    - (Optional but highly recommended) psql
+        - If you're using macOS, we recommend [using homebrew to install libpq](https://stackoverflow.com/questions/44654216/correct-way-to-install-psql-without-full-postgres-on-macos).
+2. Start the Docker daemon. You can do this by opening Docker Desktop.
```

Comment:
[dust] As mentioned in Slack, we don't currently have a way to procure licenses for Docker Desktop and it's not certified for use at the Office. Instead, my recommendations would be the following:

- Windows users: Install WSL2 and install the docker daemon inside the linux environment
- macOS users: Install Colima through homebrew, set it up, and turn it on or off using `brew services`. Install the docker compose command line tool through homebrew as well, which is allowed.
- Linux users: Install the docker daemon as normal, this is OSS